Readme.md

```diff
@@ -1 +1,78 @@
-# Discover
+# [Liquid Investigations](https://github.com/liquidinvestigations): Node
+Discovery Daemon
+
+This repository implements a method finding out in real time what other Liquid
+Investigations nodes are online.
+
+Ths discovery service must be run as `root`.
+
+
+## The basic principle
+
+[Zeroconf](https://en.wikipedia.org/wiki/Zero-configuration_networking) is
+being used for discovery of the other nodes in a network. Each node publishes
```

Comment:
Can you create a gist with example payloads and link to it, so we can use it for reference later?